thinc/layers/__init__.py

```diff
@@ -23,9 +23,9 @@
 from .tensorflowwrapper import TensorFlowWrapper
 
 # Combinators
-from .add import add
+from .add_module import add
 from .bidirectional import bidirectional
-from .chain import chain
+from .chain_module import chain
```

Comment:
I had to add the updates for `chain_module.py` and `add_module.py` although I intended to do it in a separate PR. But it was needed for the mypy plugin's tests.